app/views/docs/index.phtml

```diff
@@ -46,7 +46,14 @@ $cols = [
                     <li><a href="/docs/getting-started-for-apple">Appwrite for Apple</a></li>
                     <li><a href="/docs/getting-started-for-android">Appwrite for Android</a></li>
                     <li><a href="/docs/getting-started-for-server">Appwrite for Server</a></li>
-                    <li><a href="/docs/command-line">Appwrite CLI</a></li>
+                    <li>
+                        <a href="/docs/command-line"><i class="icon-right-dir"></i> Appwrite CLI</a>
+                        <ul class="margin-top-small margin-bottom-small">
+                            <li>&nbsp;&nbsp;<a href="/docs/command-line-commands" data-selected="/docs/command-line"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i>Commands</a></li>
+                            <li>&nbsp;&nbsp;<a href="/docs/command-line-deployment" data-selected="/docs/command-line"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i>Deployment</a></li>
```

Comment:
You can ~~fight~~ debate @eldadfux on this one. Conflicting opinions here.

Reply:
@eldadfux, since the CLI is used to manage your projects, I think the deployment content makes the most sense to introduce first. Then, the Commands are kind of a "btw, you can do this too, and here's a reference of all the commands"

Reply:
@gewenyu99 let's go with Steven suggestion and see later which page gets more traffic.